src/commands/build/wranglerjs/mod.rs

```diff
@@ -159,35 +159,54 @@ fn setup_build(target: &Target) -> Result<(Command, PathBuf, Bundle), failure::E
 
     command.arg(format!("--wasm-binding={}", bundle.get_wasm_binding()));
 
-    let webpack_config_path = PathBuf::from(
-        &target
-            .webpack_config
-            .clone()
-            .unwrap_or_else(|| "webpack.config.js".to_string()),
-    );
+    let webpack_config_path = if let Some(webpack_config) = &target.webpack_config {
+        Some(PathBuf::from(&webpack_config))
+    } else if target.site.is_none() {
+        message::warn("In Wrangler v1.6.0, you will need to include a webpack_config field in your wrangler.toml to build with a custom webpack configuration.");
```

Comment:
i think we also need to switch this warning based on whether there is a wrangler.config.js at the root (what we would use as the default).

Reply:
thanks :) + fixed